src/App.js

```diff
@@ -9,6 +9,7 @@ import SignUp from './pages/sign-up'
 import Confirmation from './pages/confirmation'
 import Item from './pages/item'
 import Checkout from './pages/checkout'
+import ContextSource from './context/root'
```

Comment:
You've imported ContextSource but you haven't used it

Reply:
wrapped the whole App return() in a <ContextProvider> tag so hopefully that's fixed.